client/src/pages/Message/SideBar/ConversationList/ConversationList.tsx

```diff
@@ -0,0 +1,29 @@
+import { Avatar, Box, Grid, List, ListItem, ListItemIcon, ListItemText, Paper, Typography } from '@material-ui/core';
+import React from 'react';
+import { Conversation } from '../../../../interface/Message';
+import AvatarIcon from '../AvatarIcon/AvatarIcon';
+import Content from '../Content/Content';
+import useStyles from './useStyles';
+
+export interface ConversationProps {
+  conversationList: Conversation[];
+}
+
+export default function ConversationList({ conversationList }: ConversationProps): JSX.Element {
+  const classes = useStyles();
+  return (
+    <Paper elevation={3} className={classes.root}>
+      <Box className={classes.sideBarContainer}>
+        <Typography className={classes.sideBarTitle}>Inbox Messages</Typography>
+        {conversationList.length
```

Comment:
Why are we checking for the conversation list length here? Can we not directly map over the conversations array?

Reply:
yeah, I think we can directly map the array